doc/interfaces/index.rst

```diff
@@ -7,5 +7,7 @@ DIPY Workflows Interfaces
 .. toctree::
 
     basic_flow.rst
+    data_fetch.rst
```

Comment:
The `data_fetch.rst` is already in the master copy so I'd say that this will create a conflict when the draft status will be removed from the PR.

Reply:
Addressed in  19378c1

Reply:
Not sure what's going on, since no conflicts are reported but the `master` does have that entry. @areeshatariq Can you rebase on master please?

Reply:
The branch has been rebased on `master`.